repair/repair.hh

```diff
@@ -323,8 +323,6 @@ struct get_sync_boundary_response {
 // Return value of the REPAIR_GET_COMBINED_ROW_HASH RPC verb
 struct get_combined_row_hash_response {
     repair_hash working_row_buf_combined_csum;
-    // The number of rows in the working row buf
-    uint64_t working_row_buf_nr;
 };
```

Comment:
I don't think this is enough. 3.1 returns a repair_hash, and that serializes differently from a struct that has one repair_hash member, because we serialize the struct size in front.

Reply:
Ah, then we need to drop the struct.